documentation/book/con-overview-distributed-tracing.adoc

```diff
@@ -0,0 +1,48 @@
+// Module included in the following assemblies:
+//
+// assembly-distributed-tracing.adoc
+
+[id='con-overview-distributed-tracing-{context}']
+= Overview of distributed tracing in {ProductName}
+
+Distributed tracing allows developers and system administrators to track the progress of transactions between applications and microservices in a distributed system. This information is useful for monitoring application performance at the micro-level and investigating issues with target systems and end-user applications.
```

Comment:
microservice-level?

Reply:
Well, they may not use micro-services! I'll just remove that description.